src/main/java/org/gol/Grid.java

```diff
@@ -20,4 +20,23 @@ private void fillGrid() {
             }
         }
     }
+
+    StringBuilder builder = new StringBuilder();
+
+    @Override
+    public String toString() {
+        builder.append("Widht = ").append(width).append("\n");
+        builder.append("Height = ").append(height).append("\n");
+        for (int i = 0; i < height; i++) {
+            for (int j = 0; j < width; j++) {
+                if (new Cell().isAlive()){
```

Comment:
@Sumlarys instead of creating a new Cell instance, you have to retrieve the cell from the cells array declared in the class using _i_ and _j_.